Dockerfile

```diff
@@ -1,17 +1,22 @@
-FROM alpine:latest
-LABEL maintainer="chris@cbrgm.de"
-LABEL version="latest"
+FROM plugins/base:amd64
 
-ARG HUGO_VERSION
+LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
+  org.label-schema.name="Drone Hugo" \
+  org.label-schema.vendor="Drone.IO Community" \
+  org.label-schema.schema-version="1.0"
 
-COPY ./drone-hugo.sh /bin/
+ARG HUGO_VERSION
+ARG HUGO_ARCH
+ENV PLUGIN_HUGO_ARCH=$HUGO_ARCH
+ENV PLUGIN_HUGO_SHIPPED_VERSION=$HUGO_VERSION
 
+ADD release/linux/amd64/drone-hugo /bin/
 RUN apk update && \
```

Comment:
Drop the update command, not needed with the line below.

Reply:
And you should also move the run step above the add step, when we start caching the layers it will stop rebuilding this step all the time